### DISCUSSION TITLE
Add additional domains that are allowed to use the talk backend server

### DISCUSSION THREAD

**Discussion by IrArevalo00** [nextcloud/all-in-one#3782] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 
I want to use the nextcloud talk aio container as the backend for another instance on a different domain.
On the talk settings, everything says _"OK"_, but when opening a call I get 

> Failed to establish signalling connection. Something might be wrong in the signalling server configuration

and on the console I see

> error: Object { code: "invalid_backend", message: "The backend URL is not supported." }

### Describe the solution you'd like
Be able to add additional domains that are allowed to use the talk backend server, via environment variables or other

### Describe alternatives you've considered
I tried modifying on the talk container the /conf/signaling.conf directly adding another backend but this, of course, does not persist a reboot and thus does not work...
 Similar to this: https://github.com/strukturag/nextcloud-spreed-signaling/issues/527

### Additional context
![image](https://github.com/nextcloud/all-in-one/assets/22783255/8a633065-97c4-49ff-ac53-89ec3e6d7bda)

**Comment by szaimen** [comment, marked as answer]
Hi, I fear such a config option is out of scope for AIO. 

However you might set up your own Talk HPB independently which will allow for this kind of configurability.

Apart from that you could set up multiple AIO instances that will have the talk hpb included for each of them: https://github.com/nextcloud/all-in-one/blob/main/multiple-instances.md
